api/auth/auth-middleware.js

```diff
@@ -30,22 +47,45 @@ const only = (role_name) => (req, res, next) => {
 
     Pull the decoded token from the req object, to avoid verifying it again!
   */
-  next();
-};
+const only = (role_name) => async (req, res, next) => {
+  try {
+    const token = req.headers.authorization;
 
-const checkUsernameExists = (req, res, next) => {
-  /*
+    console.log("role", req.decoded.role_name);
+    if (!token || req.decoded.role_name !== role_name) {
```

Comment:
cand incerci sa faci `/:user_id` ce se afiseaza in consola? Se executa acest console log?

Reply:
![image](https://user-images.githubusercontent.com/51832550/111073045-d58b8580-84e5-11eb-8dbb-d49c10c67456.png)